From 0167deb67216e3237f73facd00d268b2bd274ab6 Mon Sep 17 00:00:00 2001
From: Taylor Blau <me@ttaylorr.com>
Date: Sat, 21 Apr 2018 10:06:06 -0700
Subject: [PATCH] postpone.c: retain message_id when resending

---
 postpone.c | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/postpone.c b/postpone.c
index 7c5ca5b8..6ad823bd 100644
--- a/postpone.c
+++ b/postpone.c
@@ -575,13 +575,9 @@ int mutt_prepare_template (FILE *fp, CONTEXT *ctx, HEADER *newhdr, HEADER *hdr,
   newhdr->content->length = hdr->content->length;
   mutt_parse_part (fp, newhdr->content);
 
-  /* If resending a message, don't keep message_id or mail_followup_to.
-   * Otherwise, we are resuming a postponed message, and want to keep those
-   * headers if they exist.
-   */
+  /* If resending a message, don't keep mail_followup_to. */
   if (resend)
   {
-    FREE (&newhdr->env->message_id);
     FREE (&newhdr->env->mail_followup_to);
   }
 
-- 
2.17.0

